chain/chunks/src/lib.rs

```diff
@@ -1472,6 +1474,7 @@ impl ShardsManager {
 
         // Merge parts and receipts included in the partial encoded chunk into chunk cache
         self.encoded_chunks.merge_in_partial_encoded_chunk(partial_encoded_chunk);
+        debug!(target:"client", "merged {:?}", self.encoded_chunks.get(&header.chunk_hash()).unwrap().parts);
```

Comment:
same here - having block height helps debugging a lot

Reply:
This line was added accidentally. I will remove it